go/engine/kex2_provisionee.go

```diff
@@ -597,6 +614,33 @@ func (e *Kex2Provisionee) cacheKeys() (err error) {
 	return nil
 }
 
+func (e *Kex2Provisionee) SigningKey() (libkb.GenericKey, error) {
+	if e.eddsa == nil {
+		return nil, fmt.Errorf("provisionee missing signing key")
```

Comment:
`fmt.Errorf` implies that you need to format the string, when these are all constant strings.  Better to just skip it:  

https://golang.org/src/fmt/print.go?s=5540:5590#L194